### ISSUE TITLE
[UWP] Card throws warning in UWP but not other renderers

### ISSUE THREAD

**Issue by paulcam206** [microsoft/AdaptiveCards#2779]
```json
{
	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
	"type": "AdaptiveCard",
	"version": "1.0",
	"body": [
		{
			"type": "ColumnSet",
"style": "warning",
			"columns": [
				{
					"type": "Column",
					"width": "auto",
"style": "attention",
					"items": [
						{
							"type": "Image",
							"url": "https://pbs.twimg.com/profile_images/3647943215/d7f12830b3c17a5a9e4afcc370e3a37e_400x400.jpeg",
							"size": "small",
							"style": "person"
						}
					]
				},
				{
					"type": "Column",
					"width": "stretch",
"style": "accent",
					"items": [
						{
							"type": "TextBlock",
							"text": "Matt Hidinger",
							"weight": "bolder",
							"wrap": true
						},
						{
							"type": "TextBlock",
							"spacing": "none",
							"text": "Created {{DATE(2017-02-14T06:08:39Z, SHORT)}}",
							"isSubtle": true,
							"wrap": true
						}
					]
				}
			]
		}
	]
}
```

yields warning `Unknown properties or values inside ColumnSet - Line 7`

no warning in WPF or TS

**Comment by RebeccaAnne** [comment]
This is likely because the schema is not up to date.

**Comment by shalinijoshi19** [comment]
Likely fixed by Issue #2407

**Comment by shalinijoshi19** [comment]
Closing as dup of #2407
